guidelines/terms/22/minimum-bounding-box.html

```diff
@@ -1,6 +1,6 @@
 <dt class="new"><dfn id="dfn-bounding-boxes" data-lt="bounding boxes|bounding box">minimum bounding box</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. For components which wrap onto multiple lines as part of a sentence or block of text (such as hypertext links), the bounding box is based on how the component would appear on a single line.</p>
+   <p>the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. For components which wrap onto multiple lines as part of a sentence or <a>block of text</a> (such as hypertext links), the bounding box is based on how the component would appear on a single line.</p>
```

Comment:
There is no "block of text" definition. Looking at the code, it doesn't seem like respec is going to understand that this needs to go to "blocks of text". You'll have to adjust this.

Reply:
There is a definition, including the singular: https://github.com/w3c/wcag/blob/mjm-wcag-editorial-suggestions/guidelines/terms/20/blocks-of-text.html

Reply:
Given your thumbs up, @WilcoFiers I'm marking this as resolved. If you consider it still unresolved, please add another comment.